### PR TITLE
feat(tx): Add NFTokenID link to NFT transaction pages

### PR DESCRIPTION
**Pull request by shawnxie999**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before / After
Before
image
image
image

After
image
image
image

### CODE REVIEW

**Comment by ckniffen** [comment]
Can you use `NFTokenLink` on the NFT tab for an account too?

File: src/containers/shared/components/NFTokenLink.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export interface NFTokenLinkProps {
+  link?: boolean
```
Comment (ckniffen) [inline src/containers/shared/components/NFTokenLink.tsx:5]: Where is this set to true?

**Review by ckniffen — APPROVED** [review]

File: src/containers/shared/components/NFTokenLink.tsx (hunk 1 above)
Comment (ledhed2222) [reply src/containers/shared/components/NFTokenLink.tsx:5]: it looks like it's defaulted to true regardless. so where is it set to _false_ is the question? And why would be it be set to false?

also, this isn't a good name for an argument. `showLink` or `displayLink` would be better. Furthermore, the name of the component - `NFTokenLink` tells me that this is always a link. Why would I use this without a link?

**Review by ledhed2222 — CHANGES_REQUESTED** [review]

File: src/containers/shared/components/NFTokenLink.tsx (hunk 1 above)
Comment (shawnxie999) [reply src/containers/shared/components/NFTokenLink.tsx:5]: yeah fair, i was thinking of following the same convention used in `Account` component. but i dont think non-link will be used anywhere. I will remove it

**Review by ledhed2222 — APPROVED** [review]

**Pull request merged**